kitty/index.html

```diff
@@ -0,0 +1,155 @@
+                  <a href="#"><li><span class="icon-request"></span>Request</li></a>
+                  <a href="#"><li><span class="icon-approval"></span>Approval</li></a>
+                  <a href="#"><li><span class="icon-procure"></span>Procure</li></a>
+                  <a href="#"><li><span class="icon-receive"></span>Receive</li></a>
+                  <a href="#"><li><span class="icon-inventory"></span>Inventory</li></a>
+                  <a href="#"><li><span class="icon-payment"></span>Accounts Payable</li></a>
+                  <a href="#"><li><span class="icon-manage"></span>Manage</li></a>
+                  <a href="#"><li><span class="icon-report"></span>Reports</li></a>
+                  <a href="#"><li><span class="icon-settings"></span>Settings</li></a>     
+                </ul>
+              </nav>
+          </header>
+          <div class="container-fluid">
+
+            <div class="row">
+            
+            <div class="col-md-6  col-sm-12 col-xs-12">
+              <div class="search">
+                <span class="icon-search"></span>
+                <input type="text" name="search orders" placeholder=" What are you looking for?"> 
+              </div>
+              
+              <div class="box">
+                <h5>Pending for approvals</h5>  
+           
+                <div class="box pending-items">   
+                  <h2>Order Request</h2>
+                  <p >6 </p>
+                </div>
+
+                <div class="box pending-items">
+                  <h2>Expense Request</h2>
+                  <p>6</p>
+                </div>
+
+                <div class="box pending-items">
+                  <h2>Travel Request</h2>
+                  <p>6</p>
+                </div>
+              </div>
+
+              <div class="box">
+                <h5>Recent Comments</h5>
+                <div class="profile-comment">
+                  <div class="profile-photo">
+                    <img src="img/profilephoto.jpg"  height="50" class="clip-circle">
+                  </div>
+                </div>
+                <p>Order #123 Jul 16, 2018, 3:23pm <br> 
+                Purhased by: Cid Karmer</p>
+              </div>   
+            </div>
+           
+            <div class="col-md-6 col-sm-12 col-xs-12 ">
+                <div class="card">
+                    <div class="card-body">
+                      <h5 class="card-title">What's New?</h5>
+                      <p class="card-text">Vestibulum id ligula porta felis euismod semper. Maecenas sed diam eget risus varius blandit sit amet non magna..</p>
+                      <a href="#" class="card-link">Learn more</a>
+                     
+                    </div>
+                  </div>
+             
+              <div class="box recent-activities">
+                <h5>Recent Activities</h5>
+                    
+                <div class="recentcontainer">
```

Comment:
Nice! When we have items that are repeated like these activity items, we try to name them in such a way that we understand the relationship. So if the wrapping div has the `.recent-activities` class, each child could have a class named `.recent-activity` or simply `.activity` (be careful with the last one, if they are too general they have a higher chance of clashing with other class names)